generator/src/templates/theiaPlugins.json

```diff
@@ -31,7 +31,7 @@
     "vscode-builtin-lua": "https://open-vsx.org/api/vscode/lua/1.45.1/file/vscode.lua-1.45.1.vsix",
     "vscode-builtin-make": "https://open-vsx.org/api/vscode/make/1.45.1/file/vscode.make-1.45.1.vsix",
     "vscode-builtin-markdown": "https://open-vsx.org/api/vscode/markdown/1.45.1/file/vscode.markdown-1.45.1.vsix",
-    "vscode-builtin-markdown-language-features": "https://open-vsx.org/api/vscode/markdown-language-features/1.45.1/file/vscode.markdown-language-features-1.45.1.vsix",
+    "vscode-builtin-markdown-language-features": "https://openvsxorg.blob.core.windows.net/resources/vscode/markdown-language-features/1.39.1/vscode.markdown-language-features-1.39.1.vsix",
```

Comment:
should we keep open-vsx.org instead ?

Reply:
It would be awesome, but we can't until https://github.com/eclipse-theia/vscode-builtin-extensions/issues/41 is fixed.